src/snippet.js

```diff
@@ -0,0 +1,30 @@
+
+export default (
```

Comment:
nit: We should give this function a name, like this:
```ts
const snippet = () => { ... }

export default snippet
```

This is simply so that the runtime will produce better stack traces. You'll actually see the name `snippet` instead of `(anonymous)` if there is an error.